src/api.jl

```diff
@@ -619,6 +619,32 @@ function get_transaction(ctx::Context, id::AbstractString; kw...)
     return rsp.transaction
 end
 
+function get_transaction_events(ctx::Context, txn_id::AbstractString; kw...)
+    # TODO: support a cancellation token
+    path = PATH_ASYNC_TRANSACTIONS * "/$txn_id/events"
+    continuation_token = "0"
+    events = Channel()
+    @spawn begin
+        try
+            while continuation_token != ""
+                @info "requesting events" continuation_token txn_id
+                rsp = _get(ctx, path * "/profiler&continuation_token=$(continuation_token)"; kw...)
```

Comment:
The _get method accepts query params as an argument so no need to concatenate strings here, you can just pass it in.